WalletWasabi.Fluent/Views/Wallets/Home/History/HistoryTable.axaml

```diff
@@ -113,10 +113,56 @@
     </DataBox.Styles>
 
     <DataBox.Columns>
+      <!-- Indicators -->
+      <DataBoxTemplateColumn Width="Auto"
+                             MinWidth="120"
+                             CanUserSort="False">
+        <DataBoxTemplateColumn.CellTemplate>
+          <DataTemplate>
+            <StackPanel Orientation="Horizontal" HorizontalAlignment="Center">
+              <Button x:DataType="historyItems:HistoryItemViewModelBase" Classes="plain" ToolTip.Tip="Show Details"
+                      Command="{Binding ShowDetailsCommand}">
+                <PathIcon Data="{StaticResource details_regular}" Height="15" Opacity="0.6" />
+              </Button>
```

Comment:
Hard to notice that it is a button and it is clickable. I guess it is because it is next to two icons and it looks also like a status icon.

Reply:
experimenting with only showing it on pointer over.